test/passes/remove-unused-brs_all-features.wast

```diff
@@ -96,55 +96,4 @@
    )
   )
  )
-
- ;; a br_on of the obviously incorrect kind can just flow out the value as the
- ;; break is never taken
- (func $br_on-to-flow
-  ;; brs to func
-  (drop
-   (block $func (result (ref null func))
-    (drop
-     (br_on_func $func
```

Comment:
Should this be replaced with a test using `br_on_cast`? I don't see one in this file.

Reply:
No, `br_on_cast` is already tested in test/lit/passes/remove-unused-brs-gc.wast, so I intentionally did not replace the test in this file.